packages/jest-util/src/requireOrImportModule.ts

```diff
@@ -14,8 +14,10 @@ export default async function requireOrImportModule<T>(
   filePath: Config.Path,
   applyInteropRequireDefault = true,
 ): Promise<T> {
-  if (!isAbsolute(filePath) && filePath[0] === '.') {
```

Comment:
this is a breaking change, should probably back it out... That would also allow us to not mess with a million tests 😅